06_miscellaneous/floss_licenses_slides.md

```diff
@@ -0,0 +1,173 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+    font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+  .reveal code {
+    font-family: 'Ubuntu Mono';
+    color: orange;
+  }
+  .reveal section img {
+    background:none;
+    border:none;
+    box-shadow:none;
+  }
+</style>
+
+# FLOSS Licenses
+
+---
+
+## Disclaimer
+
+- IANAL: _I am not a lawyer._
+- Provided information "as-is", no warranty
+- Things are simplified sometimes.
+
+---
+
+## Licenses are Complicated, Right?
+
+- The details, yes
+- The big picture, no
+- Every RSE should know the big picture.
+
+---
+
+## Use Cases
+
+- You want to pick a license for your (new) code.
+- You use software of others and want to know what you are allowed to do.
+- You contribute to an existing software and want to understand what this legally means.
+
```

Comment:
```suggestion
- You contribute to an existing software and want to understand what this legally means.
- You want others to be able to use your software.
```
I would potentially add the last point to emphasize point 1 and 2 from the others perspective. I was in a course some time where they emphasized how important it is that one has a license. Otherwise adoption of new software is hard/impossible. This is especially true for businesses/companies.